guides/common/modules/proc_viewing-general-server-information.adoc

```diff
@@ -0,0 +1,19 @@
+[id="viewing-general-{project-context}-information_{context}"]
+= Viewing General {Project} Information
+
+From the *About* page in {ProjectWebUI}, you can find an overview of the following:
+
+* System Status, including {SmartProxies}, Available Providers, Compute Resources, and Plug-ins
+* Support Information
+* System Information
+* Backend System Status
+* Installed Packages
+
+To navigate to the *About* page:
+
+* In the upper right corner of {ProjectWebUI}, click *Administer* > *About*.
```

Comment:
```suggestion
.Procedure
* In the {ProjectWebUI}, navigate to *Administer* > *About*.
You can find an overview of the following:
** System Status, including {SmartProxies}, Available Providers, Compute Resources, and Plug-ins
** Support Information
** System Information
** Backend System Status
** Installed Packages
```

If this is a procedure, you'll have to add `.Procedure` followed by a list of instructions. I am unsure what others think about the list of elements. cc @adamlazik1 @Levi-Leah

Reply:
Indeed, procedure needs to be followed with a numbered list:
```
.Procedure
. Step 1
. Step 2
```

Reply:
Though, I am unsure whether this should be made into procedure since it is an enumeration of various actions. I would probably just edit the first sentence to: `From the *Administer* > *About* page in {ProjectWebUI}, you can find an overview of the following:`.
This way you can delete the `To navigate to the *About* page` section and leave the rest intact.

Reply:
@adamlazik1 @maximiliankolb I was thinking the same .I just copied the existing content and moved it under a different title, should've revised that too! Does it make sense to have the different elements as a list though?

Reply:
Well, two things:
1. I don't think this module itself is all that useful because it provides an overview of the page but no user is gonna read it. Users only read documentation when they need to accomplish something and that means they are looking for procedures.
2. If you want to include this module nonetheless, I think it does make sense to use list for different elements because it is an overview, meaning an enumeration of different objects on the page.

Reply:
@adamlazik1 makes sense, the only valuable information here is that the `Backend system status` is displayed here, which is anyway covered under `hammer ping`, so it's better to just get rid of this entire section. I'll do that and push a commit.